src/components/nav.component.astro

```diff
@@ -22,7 +23,7 @@ const imprintLink = `${import.meta.env.BASE_URL}imprint`;
     align-items: flex-start;
     justify-content: space-between;
 
-    background: var(--ddjs-color-white);
+    background: var(--ddjs-color-4);
```

Comment:
Is there such a color? If so, the name seems to be incomplete. An index should describe some kind of color scope, e.g. `primary`, `grey` or `mono`.